src/main/java/edu/hawaii/its/api/configuration/PropertiesFileOverridesReporter.java

```diff
@@ -0,0 +1,23 @@
+package edu.hawaii.its.api.configuration;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import javax.annotation.PostConstruct;
+
+@Configuration
+public class PropertiesFileOverridesReporter {
+
+    private static final Log logger = LogFactory.getLog(PropertiesFileOverridesReporter.class);
+    private final String flag;
+
+    public PropertiesFileOverridesReporter(@Value("${overrides.file.found:}") String flag) {
+        this.flag = flag;
+    }
+
+    @PostConstruct
+    public void checkOverridesStatus() {
+        logger.info("Properties overrides file flag: " + flag);
```

Comment:
We should change this logger message to be more informative. Instead of `"Properties overrides file flag: "`, we could say: `"Overrides property file found: "`

Reply:
done, thank you